app/assets/javascripts/application/welcome.js

```diff
@@ -47,10 +47,12 @@ $(document).ready(function() {
     },
     complete: function(xhr, data, status) {
       $('.progress-striped').hide();
-      if(status === "success")
+      if (status === "success")
         return $(App.Strings.successfulEmailSubmission).insertAfter(this);
+      else if (data.responseJSON.message)
+        return $(data.responseJSON.message).insertAfter(this);
```

Comment:
I think the message here needs to get wrapped in a tag? So that the jquery helper makes a new node rather than interpreting it as a selector.